src/masonite/orm/collection/Collection.py

```diff
@@ -49,9 +54,14 @@ def collapse(self):
             items += item
         return self.__class__(items)
 
-    def contains(self):
-        # todo implements this methods
-        pass
+    def contains(self, key, value=None):
+        if value:
+            return self.contains(lambda x: self._data_get(x, key) == value)
+
+        if self._check_is_callable(key, raise_exception=False):
+            return self.first(key) is not None
```

Comment:
can this just e `return self.first()`?

Reply:
In this case we need to return boolean.
I think self.first() is not the good response

Reply:
Ok